Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -1,33 +1,39 @@
-# Rendering in JavaScript
+# Common Rendering and JVM
 
 It is always good to question ourselves - do we really need the
 JVM backend for the project or not. What if we would be able
-to do the [Mandelbrot Set](https://en.wikipedia.org/wiki/Mandelbrot_set)
+to run the [Mandelbrot Set](https://en.wikipedia.org/wiki/Mandelbrot_set)
 rendering on the client? 
 
 Let's implement the client-side rendering and see how it goes.
 
 ## Sharing Code
 
 We've just added the very first Kotlin file into the `commonMain`
```

Comment:
We've just added our first Kotlin file into the `commonMain`
source set. Let's move all the files from the `jvm` source set
into the `common` source set so we share the code with JavaScript.